opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/SpanDecorator.java

```diff
@@ -44,15 +45,27 @@
 
     /**
      * Decorate span when an exception is thrown during processing in
-     * {@link javax.servlet.Filter#doFilter(ServletRequest, ServletResponse, FilterChain)}.
-     * It can be {@link RuntimeException} from {@link javax.servlet.Filter} or any checked exception from
-     * {@link javax.servlet.http.HttpServlet}.
+     * {@link javax.servlet.Filter#doFilter(ServletRequest, ServletResponse, FilterChain)}. This is
+     * also called in {@link javax.servlet.AsyncListener#onError(AsyncEvent)}.
      *
      * @param httpServletRequest request
      * @param exception exception
      * @param span span to decorate
      */
-    void onError(HttpServletRequest httpServletRequest, Throwable exception, Span span);
+    void onError(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse,
```

Comment:
It is released but still under 0.xx. I would rather do breaking changes and once this is more stable release 1.x..